app/controllers/photos_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PhotosController < ApplicationController
+  def index
+    @photos = Photo.all.order(created_at: :desc)
+  end
+
+  def show
+    @photo = Photo.find(params[:id])
+    @average = @photo.avg
```

Comment:
I'd avoid assigning extra instance vars - just reference the attribute in the view.